site_scons/components/__init__.py

```diff
@@ -236,7 +234,7 @@ def define_mercury(reqs):
                           ['make', 'install']],
                 libs=['mercury', 'na', 'mercury_util'],
                 pkgconfig='mercury',
-                requires=[atomic, 'boost', 'ofi'] + libs,
+                requires=[atomic, 'boost', 'ofi', 'ucx'] + libs,
```

Comment:
I'm not sure what this means for mercury... Isn't mercury either ofi or ucx, but not both? Maybe I have misunderstood this.